### DISCUSSION TITLE
Welcome to docker_flutter Discussions!

### DISCUSSION THREAD

**Discussion by PlugFox** [PlugFox/docker_flutter#25] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PlugFox** [comment]
@DoumanAsh @dkrutskikh @MariaMelnik @zs-dima 
I'm planning:
+ merge `android` and `android warmed` into 1 image
+ replace `flutter` user with `root`
+ run flutter doctor inside a base image

What do you think about that?

**Reply by PlugFox** [reply]
Also, maybe somebody can help me with glibc for arm image?

**Reply by zs-dima** [reply]
@PlugFox

>    merge android and android warmed into 1 image

    Do you mean you think to remove android non-warmed image?


>    replace flutter user with root

    in case of android I do not expect any issue.
    however for the web - flutter user looks more secure.


>    run flutter doctor inside a base image

    flutter doctor looks more relevant for the resulting image.
    are there any advantages to run flutter doctor inside a base image?

**Reply by PlugFox** [reply]
> Do you mean you think to remove android non-warmed image?

Yeap, merge it in one

> however for the web - flutter user looks more secure.

What the difference?

> are there any advantages to run flutter doctor inside a base image

Yeap, base imaged used for unit tests and analyzer.
And they run flutter doctor everytime.
We can speedup that process by preaching In base image.

**Reply by DoumanAsh** [reply]
> merge android and android warmed into 1 image

Yes, I think it is fine

> replace flutter user with root

That's good I think.
I don't know why flutter always complains about running under `root` but it is pointless to create separate user for `flutter` anyway

> run flutter doctor inside a base image

I generally prefer base image to be shell.
It is convenient to just run image and test its capabilities?
